activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -329,13 +329,13 @@ def initialize(spec)
         # default max pool size to 5
         @size = (spec.config[:pool] && spec.config[:pool].to_i) || 5
 
-        # The cache of threads mapped to reserved connections, the sole purpose
-        # of the cache is to speed-up +connection+ method, it is not the authoritative
-        # registry of which thread owns which connection, that is tracked by
+        # The cache of threads mapped to reserved connections. The sole purpose
+        # of the cache is to speed-up +connection+ method. It is not the authoritative
+        # registry of which thread owns which connection. That is tracked by
```

Comment:
Here also it reads a bit awkward, may be combine with earlier statement from  https://github.com/rails/rails/pull/25717/files#r69738614?